commands/report.js

```diff
@@ -24,7 +24,7 @@ exports.run = async (client, message, args) => {
     return message.channel.send(`${redtick} Please specify an user and a reason`)
 
   const user = client.utils.parseMention(args.shift())
-  if(message.author.id === user) return message.reply("You shoudn't report yourself <:kappalul:527037429161525248>")
+  if(message.author.id === user) return message.reply(`${redtick}You shoudn't report yourself`")
```

Comment:
Add a space between the redtick and the text

Remove the double quote at the end

It's shou**l**dn't, not *shoudn't*

Reply:
how i can do thisthose errors